core/services/vrf/v2/listener_v2_log_listener.go

```diff
@@ -0,0 +1,451 @@
+package v2
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"math"
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/pkg/errors"
+	"go.uber.org/multierr"
+
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/logpoller"
+	evmtypes "github.com/smartcontractkit/chainlink/v2/core/chains/evm/types"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/chainlink/v2/core/services/pg"
+	"github.com/smartcontractkit/chainlink/v2/core/services/vrf/vrfcommon"
+	"github.com/smartcontractkit/chainlink/v2/core/utils"
+	"github.com/smartcontractkit/chainlink/v2/core/utils/mathutil"
+)
+
+func (lsn *listenerV2) runLogListener(
+	pollPeriod time.Duration,
+	minConfs uint32,
+) {
+	lsn.l.Infow("Listening for run requests via log poller",
+		"minConfs", minConfs)
+	ticker := time.NewTicker(utils.WithJitter(pollPeriod / 2))
```

Comment:
is it ok that the jitter will bump the ticker above period/2 and therefore lead to aliasing in some cases?